client/my-sites/themes/single-site.jsx

```diff
@@ -94,6 +98,11 @@ const ThemesSingleSite = React.createClass( {
 						hideForTheme: theme => ! isPremium( theme )
 					}
 					: {},
+				help: ! site.jetpack // We don't know where support docs for a given theme on a self-hosted WP install are.
```

Comment:
The comment (about "support docs") is a bit misleading here, since we're using `getHelpUrl` to point to forum resources (as opposed to setup docs, which is what `getSupportUrl` is about -- and where this comment is from). So to differentiate, this comment should be about "support forums".